dss.tf

```diff
@@ -0,0 +1,119 @@
+{
```

Comment:
I would recommend using HCL not JSON, as it is much more human readable + editable.